hub/static/js/area.esm.js

```diff
@@ -40,6 +40,20 @@ function updateUI() {
     })
 }
 
+async function toggleFavourite($form) {
+    const response = await fetch($form.attr('action'), {
+        method: $form.attr('method') || 'GET',
+        mode: 'cors',
+        credentials: 'same-origin',
+        body: $form.serialize(),
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
Would be nice to standardise on either single or double quotes around strings – looks like we mostly use single quotes in this file?